phalcon/paginator/adapter/querybuilder.zep

```diff
@@ -167,7 +167,11 @@ class QueryBuilder extends Adapter implements AdapterInterface
 		 */
 		var groups = totalBuilder->getGroupBy();
 		if !empty groups {
-			var groupColumn = implode(", ", groups);
+			if is_array(groups) {
+				let groupColumn = implode(", ", groups);
```

Comment:
```php
if typeof groups == "array" {
	var groupColumn = implode(", ", groups);
} else {
	var groupColumn = groups;
}
totalBuilder->groupBy(null)->columns(["COUNT(DISTINCT ".groupColumn.") AS rowcount"]);
```

Zephir\CompilerException: Cannot read variable 'groupColumn' because it wasn't declared in /home/MKas/Documents/sources/cp/cphalcon/phalcon/paginator/adapter/querybuilder.zep on line 175

totalBuilder->groupBy(null)->columns(["COUNT(DISTINCT ".groupColumn.") AS rowcount"]);